mne/data/coil_def_Elekta.dat

```diff
@@ -39,6 +39,12 @@
 #			2	accurate
 #
 #
+1   2000    2   1  0.000e+00  0.000e+00	"Point magnetometer, x-normal"
```

Comment:
Maybe we're talking about different files... In the original `coil_def.dat` file:
>1   2000    0   1  0.000e+00  0.000e+00	"Point magnetometer"
> 1.0000  0.000e+00  0.000e+00  0.000e+00  0.000  0.000  1.000

`coil_def_elekta.dat` has them defined differently (2000 is x, 2001, is y, 2003 is z), but it should probably be consistent with `coil_def.dat`, right?